pkg/testing/fixture_install.go

```diff
@@ -16,10 +16,15 @@ import (
 	"runtime"
 	"strconv"
 	"strings"
+	gotesting "testing"
 	"time"
 
+	"github.com/stretchr/testify/assert"
 	"github.com/stretchr/testify/require"
 
+	"github.com/elastic/elastic-agent-libs/mapstr"
+	"github.com/elastic/elastic-agent-libs/opt"
+	eamprocess "github.com/elastic/elastic-agent-system-metrics/metric/system/process"
```

Comment:
Why the `m` in `eaMprocess`? If it'd to abbreviate _elastic-agent-system-metrics_ I'd suggest to include the _system_ as well
```suggestion
	easmprocess "github.com/elastic/elastic-agent-system-metrics/metric/system/process"
```
However I think something like `systemprocess` or `agentsystemprocess` is better. For me, those abbreviations always look cryptic at first glance.

Reply:
Went with `agentsystemprocess` 👍